### DISCUSSION TITLE
blog/2022/orcid-auto-update/

### DISCUSSION THREAD

**Discussion by giscus** [choldgraf/choldgraf.github.io#39] [Blog comments]
# blog/2022/orcid-auto-update/

For a while I’ve had a hand-crafted.bibtex file stored locally for my publications/ page. However, manually updating local text file is a pain to remember, especially since there are many services ...

https://chrisholdgraf.com/blog/2022/orcid-auto-update/

**Comment by bendichter** [comment]
Awesome! Now can you get this to auto-update using a GitHub workflow? 🤔

**Reply by enryH** [reply]
Currently it is not done. If you do not want to commit anything, it can be simply a script execution in the current workflow.

Really neat trick with ORCID! I had to make small adjustments as a consortium does not have a family or given name... but besides that it still works!
